app/src/main/java/org/oppia/app/walkthrough/end/WalkthroughFinalFragment.kt

```diff
@@ -22,17 +22,28 @@ class WalkthroughFinalFragment : InjectableFragment() {
       return storyFragment
     }
   }
-  @Inject lateinit var walkthroughFinalFragmentPresenter: WalkthroughFinalFragmentPresenter
+
+  @Inject
+  lateinit var walkthroughFinalFragmentPresenter: WalkthroughFinalFragmentPresenter
```

Comment:
Ditto.

Reply:
Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting